docs/usage.md

```diff
@@ -118,11 +118,121 @@ directory.
+
+For example to analyze `tmux` project daily.
+1. Generate a new log file
+```
+ CodeChecker log -b "make" -o compilation.json
+```
+3. Re-analyze the code
+```
+ CodeChecker analyze compilation.json ./reports-daily
+```
+4. Store the analysis results into the central database.
+```
+ CodeChecker store ./reports --host codechecker.central --port 8555 --name tmux_master_2017_08_28
+```
+
+and the next day repeat **Step 1** to **Step 4** and store the results under run name `tmux_master_2017_08_29`.
+
+Then you can query newly introduced bugs in the following way.
+```
+ CodeChecker cmd diff -b tmux_master_2017_08_28 -n tmux_master_2017_08_29  --new --host codechecker.central --port 8555
+```
+If you would like to generate a web page out of this, you can get the results in `json` format too:
+```
+ CodeChecker cmd diff -b tmux_master_2017_08_28 -n tmux_master_2017_08_29  --new --host codechecker.central --port 8555 -o json
+```
+
+*Note:* Delete old runs you don't need to save database space.
+
+### Storing the results of each commit and guarding the introduction of new bugs
+Let us assume that at each commit you would like to keep your analysis 
+results up-to-date and send an alert email to the programmer if a new
+bug is to be introduced in a "pull request". 
+Also when there is a new bug in the uploaded code, reject the "pull request".
+
+A single run should be used to store the analysis results of module on a specific branch: 
+`<module_name>_<branch>`
+
+The run should be always updated when a new commit is merged 
+to reflect the analysis status of the latest code version on your branch.
+
+Let's assume that user `john_doe` changed curl/src/slist_wc.c in tmux. 
+The CI loop re-analizes tmux project and sends an email with reject if new bug was found compared to the master version, or
+accepts and merges the commit if no new bugs were found. 
+
+Let's assume that the working directory is `tmux` containing John Doe's modifications. 
+
+1. Generate a new log file for the new code
+```
+ CodeChecker log -b "make" -o compilation.json
+```
+3. Re-analyze the changed code of john_doe
+```
+ CodeChecker analyze compilation.json ./reports-john-doe
+```
+4. Check for new bugs in the run
+```
+ CodeChecker cmd diff -b tmux_master -n ./reports-john-doe --new --host codechecker.central --port 8555
+```
+
+If new bugs were found, reject the commit and send an email with the new bugs to John.
+
+If no new bugs were found:
+
+5. Merge the changes into the master branch
+	
+6. Update the analysis results according to the new code version:
+```
+ CodeChecker store ./reports-john-doe --host codechecker.central --port 8555 --name tmux_master
+```
+
+If John finds a false positive report in his code and so the CI loop would prevent the merge of his pull request,
+he can suppress the false positive by amending the following suppression commend in his code a line above the bug
+```
+// codechecker_suppress [all] suppress all checker results
```

Comment:
Suppressing a specific checker is better I think. This could hide potential later findings by other checkers.